cmd/bundle/main_test.go

```diff
@@ -11,6 +11,9 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
```

Comment:
Nit: I do not think we have written rules for this, but the non-dot imports usually come after the dot imports.